src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            lottery.getRandomBall();
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```

we should print info about ball to console

Reply:
We print this information because have sout in getRandomBall() method

Reply:
> We print this information because have sout in getRandomBall() method

Привіт) Ментори не бачать цих твоїх питань в коментарях на Github, якщо в тебе є якісь уточнюючі питання по ревью, краще задачати їх в Slack в чатику jv_chat, і тегати ментора, який робив ревью 🙂